functions/ext-storage.bash

```diff
@@ -40,10 +40,25 @@ move_root2usb() {
   if ! [ $rootonsdcard = true ]; then
     infotext="It seems as if your system root is not on the SD card.
        ***Aborting, process cant be started***"
-    whiptail --title "System root not on SD card?" --msgbox "$infotext" 8 78
+    whiptail --title "System root not on SD card ?" --msgbox "$infotext" 8 78
     return 0
   fi
 
+  #exit if destination is not available
+  if ! [[ -b "$NEWROOTPART" ]]; then
+    infotext="Seems there is no external SD card inserted that we could move openHABian root to.\\n***Aborting, process cant be started***"
```

Comment:
"external SD card" would be better said as "external storage medium"

Reply:
done